charts/microgateway/values.yaml

```diff
@@ -60,7 +60,7 @@ config:
     # :exclamation: Never use `trace` in production.
     logLevel: info
     # config.global.redisService -- List of Redis services.
-    # @default -- - `redis-master`<br> If `redis.enabled=true`<br><br> - `""`<br> If `redis.enabled=false`
+    # @default -- []
```

Comment:
wrong is a strong word, it depends on the philosophy applied here. In my opinion we should not document magic internas here.
In my opinion this directive is to be used to configure an external Redis service. if a the internal Redis service is used, the end user has not to care about what the internal value of the used Redis service is.

Reply:
Ok, maybe it could be viewed from different perspective.... There is a big but!

There are a few other "default" values where the internal magic is described. At least all default values should be described the same way (either do not describe the magic or do it). At the moment it would not be consistent. And this is really bad in my point of view...

I vote to describe the internal magic for redisService

Reply:
It's a meta information that describe the default value **or** behaviour. From that point of view, we should describe how the containers are connected.

Reply:
ok, i've reverted it